src/Sparrow.Server/ByteString.cs

```diff
@@ -1597,10 +1593,7 @@ public InternalScope From(ReadOnlySpan<byte> value, int size, ByteStringType typ
             Debug.Assert(value != null, $"{nameof(value)} cant be null.");
 
             str = AllocateInternal(size, type);
-            fixed (byte* ptr = value)
-            {
-                Memory.Copy(str._pointer->Ptr, ptr, size);
-            }
+            value[..size].CopyTo(new Span<byte>(str._pointer->Ptr, size));
```

Comment:
Why do not get sub-span instead of range? 
Simple bench:
https://gist.github.com/maciejaszyk/8a6233126c75deb36909a65a44e07489

Reply:
That's unexpected. They are supposed to be the same, I would open a performance bug on CoreCLR repo.

Reply:
Done